### PR TITLE
Apply gamma LUT when an output re-enabled

### PR DESCRIPTION
**Pull request by emersion**
References: https://gitlab.freedesktop.org/wlroots/wlroots/-/issues/3679

### CODE REVIEW

File: sway/desktop/output.c [hunk 1]
```diff
@@ -961,6 +961,11 @@ static void handle_commit(struct wl_listener *listener, void *data) {
 		wlr_damage_ring_set_bounds(&output->damage_ring, width, height);
 		wlr_output_schedule_frame(output->wlr_output);
 	}
+
+	// Next time the output is enabled, try to re-apply the gamma LUT
+	if ((event->committed & WLR_OUTPUT_STATE_ENABLED) && !output->wlr_output->enabled) {
```
Comment (kennylevinsen) [inline sway/desktop/output.c:966]: Heh, slightly confusing: We're in a post commit handler where sway output is *enabled*, but the wlr_output has been *disabled*...

But yeah, sway only changes its own state *after* committing the `wlr_output`, so it makes sense.

**Review by kennylevinsen — APPROVED** [review]
LGTM

**Comment by Artturin** [comment]
Hello, this caused

```
       > ../sway/desktop/output.c: In function 'handle_commit':
       > ../sway/desktop/output.c:969:19: error: 'struct wlr_output_event_commit' has no member named 'committed'
       >   969 |         if ((event->committed & WLR_OUTPUT_STATE_ENABLED) && !output->wlr_output->enabled) {
       >       |                   ^~
 ```
 
 https://github.com/nix-community/nixpkgs-wayland/actions/runs/6945817875/job/18896048025#step:5:387
 
 `wlroots` is at `dbe7fb70273ed1c9008bb5d36546d0d187f1217c` ([newest](https://gitlab.freedesktop.org/wlroots/wlroots/-/commits/master?ref_type=heads))

**Comment by emersion** [comment]
Ah, classic rebase issue. Fixed in https://github.com/swaywm/sway/commit/9d666a08e1ec77680890a08f63e71ce6bb53b9a9.

**Comment by Artturin** [comment]
Thanks

**Pull request merged**